src/model.rs

```diff
@@ -835,26 +836,29 @@ impl Collections {
             let prev_vj = &self.vehicle_journeys[*prev_vj_idx];
             let next_vj = &self.vehicle_journeys[*next_vj_idx];
             if prev_vj.block_id != next_vj.block_id {
-                // we can discard two vehicle journeys which are not part of the
-                // same prolongation trip.
+                // we can discard when two vehicle journeys are not in a stay-in
+                // situation.
                 continue;
             }
             let last_stop = &prev_vj.stop_times.last().unwrap();
             let first_stop = &next_vj.stop_times.first().unwrap();
             if last_stop.stop_point_idx == first_stop.stop_point_idx {
-                // We can discard prolongation trip when the stop point is
-                // identical (see Example 1 above).
+                // We can when the stop points are identicals (see Example 1
```

Comment:
```suggestion
                // We can discard when the stop points are identicals (see Example 1
```

Reply:
Done in 0e5e40408a6ab5201738c7c4c344dde909a9ac04.